### PR TITLE
Add contact form and footer sections

### PR DESCRIPTION
**Pull request by thasipablo**
What added:
- Create the contact form
- Create the footer form
- Connect the form to Formspree

### CODE REVIEW

**Review by Cabraham1 — COMMENTED** [review]
# INVALID ❌ 

Hi @thasipablo    

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😓

## Invalid Code Review Request ❌

- [ ] Unfortunately, you have submitted a pair programming project but has not commit from your partner 

_**Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready**_ 🤝🙏

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — COMMENTED** [review]
Hi @Team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
You have submitted a project, that has not gitignore file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## 

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -208,5 +208,52 @@ <h3 class="about-me-card__title card-title">Skills</h3>
...
+      <!-- contact -->
+      <section id="contact" class="contact-section">
+        <div class="section-container">
+          <h1 class="section-title">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+          </h1>
+          <form method="POST" action="https://formspree.io/f/xoqzgjwv">
+            <input type="text" placeholder="Full name" name="name" maxlength="30" required>
+            <input type="email" placeholder="Email address" name="email" required>
+            <textarea cols="30" rows="10" name="message" placeholder="Hey I’d like to connect to..." required></textarea>
```
Comment (abdallahmalima) [inline index.html]: - 🙏Please add the validation for the `maxlength` attribute to the `<textarea>` element with the value `maxlength="500"`. This is a project requirement aimed at limiting 😵 the number of characters a user can enter within the `textarea` to a maximum of 500 characters. By enforcing this constraint, we ensure that the entered text does not exceed the specified limit.

**Review by abdallahmalima — CHANGES_REQUESTED** [review]
Hi @thasipablo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
- Form action and method✔️
- Form fields input type✔️
- Use [formspree service](https://formspree.io/html)✔️
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._✔️

**Review by wayungi — APPROVED** [review]
Hi @thasipablo , 

image

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] max-length validation added ✔️ 
- [ ]  Email field is validated ✔️
- [ ] name field maxlength added ✔️   


## Optional suggestions
- [ ] Please stick to GitHub flow or Git flow according to the project & name your branches accordingly> Sticking to the standard conventions simplifies collaboration ans usage of your project by other developers. 👍🏾 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**